app/components/userSummary.tsx

```diff
@@ -122,6 +115,24 @@ const UserSummary: React.FC<UserSummaryProps> = ({ assignment, selectedUser, pro
 		}
 	};
 
+	const handleDeleteAssignmentClick = () => {
+		if (assignment.canBeDeleted === true) {
```

Comment:
use just  if(assignment,caBeDeleted) instead of comparison.

Reply:
👍